site/pages/gcweb-theme/release/v5.0-en.hbs

```diff
@@ -13,281 +13,512 @@
+			<li><a href="../../tbl-floatcolumn-en.html">Float column in a data table</a></li>
+			<li><a href="../../redacted-fr.html">Redacted text</a></li>
 		</ul>
-
-		<p>Évaluation et rapport:</p>
+		<p>Assessment and report:</p>
 		<ul>
-			<li>16 november 2018: <a href="../../evaluation-report/2-wetplugin-gcweb2.html" hreflang="en" lang="en">2 - Regression user acceptance testing of WET plugin for GCWeb theme version 2</a> (disponible en anglais seulement)</li>
-			<li>14 novembre 2018: <a href="../../evaluation-report/1-accessibility.html" hreflang="en" lang="en">1 - Accessibility assesment of GCWeb theme version 2</a> (disponible en anglais seulement)</li>
+			<li>November 16, 2018: <a href="../../evaluation-report/2-wetplugin-gcweb2.html" hreflang="en" lang="en">2 - Regression user acceptance testing of WET plugin for GCWeb theme version 2</a></li>
+			<li>November 14, 2018: <a href="../../evaluation-report/1-accessibility.html" hreflang="en" lang="en">1 - Accessibility assesment of GCWeb theme version 2</a></li>
 		</ul>
-
 		<p>Notes:</p>
 		<ul>
-			<li>Le guide de rédaction pour canada.ca a été mis à jour afin de permettre le texte en italique pour certain cas d'utilisation.</li>
+			<li>The Canada.ca Content Style Guide has been updated in order to allow the use of italics in certain usage cases.</li>
 		</ul>
 	</section>
 	<section>
-		<h3>Fureteur supporté</h3>
-		<p>Tel que défini par la <a hreflang="en" href="http://wet-boew.github.io/wet-boew-documentation/decision/2.html">Décision de conception 2: Fureteur supporté</a> (en anglais)</p>
-		<ul>
-			<li>Chrome 71</li>
-			<li>Chrome 70</li>
-			<li>Safari 12.0</li>
-			<li>Firefox 65</li>
-			<li>Firefox 64.0.2</li>
-			<li>Firefox ESR - 60.5.0</li>
-			<li>IE 11</li>
-			<li>Edge 18</li>
-		</ul>
-	</section>
-</section>
-
-<section>
-	<h2 id="details">Détails</h2>
-	<p><strong>Nombre de contributions:</strong> 119</p>
-	<ul>
-		<li><a href="https://github.com/duboisp">Pierre Dubois (@duboisp)</a>: 90 contributions</li>
-		<li><a href="https://github.com/EricDunsworth">Eric Dunsworth (@EricDunsworth)</a>: 8 contributions</li>
-		<li><a href="https://github.com/patlaj">Patrick Lajeunesse (@patlaj)</a>: 7 contributions</li>
-		<li><a href="https://github.com/ipaksc">Ilya Pak (@ipaksc)</a>: 4 contributions</li>
-		<li><a href="https://github.com/shawnthompson">Shawn Thompson (@shawnthompson)</a>: 3 contributions</li>
-		<li><a href="https://github.com/anishkny">Anish Karandikar (@anishkny)</a>: 2 contribution</li>
-		<li><a href="https://github.com/Borges-Porteous">Borges-Porteous (@Borges-Porteous)</a>: 2 contribution</li>
-		<li><a href="https://github.com/GormFrank">Francis Gorman (@GormFrank)</a>: 1 contribution</li>
-		<li><a href="https://github.com/jkshapiro">Kivi Shapiro (@jkshapiro)</a>: 1 contribution</li>
-		<li><a href="https://github.com/jmealing">Jennifer Mealing (@jmealing)</a>: 1 contribution</li>
-		<li>Co-contributeur:
+		<div lang="fr">
```

Comment:
Can you do the same as you did for the wet-boew v4.0.30 release note. Take the translation from a preceding release. Thanks